scalableminds/webknossos
# Add "missing dataset"-button to the spotlight view

This PR adds a button to the spotlight view (used for demo instance) that refers to a google doc form. It gives new users/researches the possibility to contact SCM to add their dataset to the demo instance.

### URL of deployed dev instance (used for testing):
- [https://addmissingdatasetbutton.webknossos.xyz](https://addmissingdatasetbutton.webknossos.xyz)

### Steps to test:
- open the wk-demo instance and log out
- have a look at the "missing dataset"-button
- clicking on the button should open the google form in another tab
- after clicking the button the text of the button should not be highlighted or underlined.

### Issues:
- fixes #3514

------
